Formula/o/ocicl.rb

```diff
@@ -0,0 +1,48 @@
+class Ocicl < Formula
+  desc "OCI-based ASDF system distribution and management tool for Common Lisp"
+  homepage "https://github.com/ocicl/ocicl"
+  url "https://github.com/ocicl/ocicl/archive/refs/tags/v2.3.4.tar.gz"
+  sha256 "be31c1d5de3352bf2ddc759153abc9c04ce4c0b962cc1d9d07888379e18ff202"
+  license "MIT"
+
+  depends_on "oras"
+  depends_on "sbcl"
+  depends_on "zstd"
+
+  def install
+    mkdir_p libexec
+    mkdir_p bin
+
+    system "sbcl", "--dynamic-space-size", "3072", "--no-userinit", "--eval",
+           "(require 'asdf)", "--eval", <<~LISP
+             (progn
+               (push (uiop:getcwd) asdf:*central-registry*)
+               (asdf:load-system :ocicl)
+               (sb-ext:save-lisp-and-die "#{libexec}/ocicl.core"))
+           LISP
+
+    # Write a shell script to wrap oras
+    (bin/"ocicl").write <<~EOS
+      #!/usr/bin/env -S sbcl --core #{libexec}/ocicl.core --script
+      (ocicl:main)
+    EOS
+
+    # Write a shell script to wrap oras
+    (bin/"ocicl-oras").write <<~EOS
```

Comment:
Why do we need this script?

Reply:
Upstream ocicl (of which I am the author) invokes ocicl-oras, which is a vendored distribution of oras, in order to isolate it from changes to the system oras.  Oras was still new and I was concerned about oras changes that would break ocicl.  I don't think I need to do this anymore, and one day I'll remove the vendored oras from upstream.  The first step here is to use the system oras, but wrap it in a script until I change upstream.